app/controllers/appointment_controller.js

```diff
@@ -0,0 +1,612 @@
+//  === Requirements ===
+const { fake } = require('faker')
```

Comment:
Why is this added? This seems unnecessary.

Reply:
removed